templates/rustdoc/body.html

```diff
@@ -33,19 +33,18 @@
                             <ul class="pure-menu-list menu-item-divided">
                                 <li class="pure-menu-heading" id="crate-title">
                                     {{ krate.name }} {{ krate.version }}
-                                    <i class="fa fa-copy fa-1" id="clipboard" aria-label="Copy crate name and version information"></i>
+                                    {{ "copy" | far(extra='id="clipboard" aria-label="Copy crate name and version information"') | safe }}
```

Comment:
You forgot to remove the `| safe` here. Also, this needs to have `aria-hidden="false"`, since we want screen readers to see this icon (since we added a `aria-label` attribute).